docs/glossary.rst

```diff
@@ -0,0 +1,13 @@
+.. currentmodule:: sunkit_instruments
+
+.. _glossary:
+
+********
+Glossary
+********
+
+.. glossary::
+   :sorted:
+
+   temperature response
```

Comment:
What do we want to include?

* temperature response?
* response function?
* gain?